applications/osb-portal/src/pages/RepositoryPage.tsx

```diff
@@ -186,14 +189,25 @@ const useStyles = makeStyles((theme) => ({
         flexGrow: 1,
       },
       "& .preview-box": {
+        backgroundColor: "rgba(255, 255, 255)",
+        color: "rgba(0, 0, 0)",
+        paddingLeft: "2rem",
+        paddingRight: "2rem",
+        fontFamily: font,
         borderRadius: 2,
         boxShadow: "0 0 0 3px rgba(0, 0, 0, 0.25)",
-        border: `1px solid ${paragraph}`,
         minHeight: "15rem",
         overflow: "auto",
         [theme.breakpoints.up("sm")]: {
           height: "calc(100vh - 14.5rem)",
```

Comment:
It's not usually a good idea to set the height of a dom element. Flexbox capabilities should give the behavior needed here.

Reply:
idk why it was set before, maybe just a temporary setting for the empty field. @vidhya-longani